lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ApplyCoefficientsForEnsembleCalibration.py

```diff
@@ -768,14 +770,16 @@ def test_too_many_coefficients(self):
                 predictor_cube, variance_cube, optimised_coeffs,
                 coeff_names, predictor_of_mean_flag)
 
-    def test_missing_date(self):
+    @ManageWarnings(
+        record=True,
```

Comment:
Why do we need record=True here?

Reply:
This is a bit strange. If I get ManageWarnings to ignore the warning message "Ensemble calibration not available" in test_missing_date it then doesn't raise the warning message in test_missing_date_catch_warning and the checks fail. However if I change the name of the test to test_catch_warning_missing_date it then correctly raises the warning and everything works.  For now I've removed record=True, added the warning to ignored_messages and renamed the subsequent test. I'm not sure the decorator is working quite as one would hope.